extra-dev/packages/coq-compcert/coq-compcert.8.8.dev/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
```

Comment:
The CI seems to [complain](https://travis-ci.org/coq/opam-coq-archive/jobs/439271338#L905) on 2.0. Shall new packages use 1.2 or 2.0?

Reply:
For now the repos is 1.2